packages/ui-progress/src/ProgressBar/index.tsx

```diff
@@ -134,6 +134,10 @@ class ProgressBar extends Component<ProgressBarProps> {
             aria-valuemax={valueMax}
             aria-label={labelAndValueText}
           />
+
+          <span css={styles?.track} role="presentation" aria-hidden="true">
```

Comment:
We are confident, because I submitted an a11y audit yesterday and Jes already tested it and gave the green light :) I also tested it myself in 3 browsers, the tests and the VRT is working good too (even caught an error in the former version).

Reply:
@matyasf what do you think?

Reply:
i think it shoule be fine, this is purely a visual element